README.md

```diff
@@ -53,40 +34,23 @@ git branch -u upstream/main main
   <span> · </span>
   <a href="https://qiskit.org/advocates/">Advocates</a>
   <span> · </span>
-  Support: <a href="http://qisk.it/join-slack">Slack</a>
+  <a href="https://qiskit.org/ecosystem/">Ecosystem</a>  
 </h3>
 
-## What’s In This Document
+<br/>
+
+## Table of Contents
 
-- [What’s In This Document](#whats-in-this-document)
-- [⚡️ Live](#️-live)
-- [💻 Technology Used](#-technology-used)
 - [🚀 Get Up and Running](#-get-up-and-running)
+- [💻 Technology Used](#-technology-used)
 - [🏭 Content Generation](#-content-generation)
-- [🧐 Directory structure](#-directory-structure)
+- [📁 Directory structure](#-directory-structure)
 - [✏️ How to Contribute](#️-how-to-contribute)
 - [🗓 Open backlog](#-open-backlog)
 - [👩‍💻 Maintainers](#-maintainers)
+- [🔗 Connect With Qiskit](#-connect-with-qiskit)
```

Comment:
Should "With" be capitalised here? My guess is that if the "to" in "How to Contribute" and the "and" in "Get Up and Running" aren't, the "with" here shouldn't either.

Reply:
Removed in https://github.com/Qiskit/qiskit.org/pull/3172/commits/e663ede4b578cbd2c4034f7a29d7e5b5588449d1